packages/i18n/src/locales/en/options/brian.yml

```diff
@@ -6,6 +6,10 @@ armholeDepthFactor:
   title: Armhole depth factor
   description: Controls the depth of the armhole. Higher values make a deeper armhole.
 
+armholeHeight:
+  title: Armhole height
+  description: Controls the height of the armhole. Higher values result in a more tight armhole.
```

Comment:
My understanding is that this is meant to control the bottom point of the armhole, given that a higher value (which I assume means 'go up more') means a tighter armhole.

However when I read `armholeHeight` I immediately think 'how tall the armhole is' which gets confusing when reading that a higher value means a smaller armhole.